sections/futures/UserInfo/OpenOrdersTable.tsx

```diff
@@ -92,101 +95,145 @@ const OpenOrdersTable: React.FC = () => {
+											{cellProps.row.original.market}
+											{cellProps.row.original.isStale && (
+												<ExpiredBadge>
+													{t('futures.market.user.open-orders.badges.expired')}
+												</ExpiredBadge>
+											)}
+										</StyledText>
+										<StyledValue>{cellProps.row.original.orderType}</StyledValue>
+									</MarketContainer>
+								);
+							},
+							sortable: true,
+							width: 50,
+						},
+						{
+							Header: (
+								<StyledTableHeader>
+									{t('futures.market.user.open-orders.table.side')}
+								</StyledTableHeader>
+							),
+							accessor: 'side',
+							Cell: (cellProps: CellProps<any>) => {
+								return (
+									<div>
+										<PositionType side={cellProps.row.original.side} />
+									</div>
+								);
+							},
+							sortable: true,
+							width: 50,
+						},
+						{
+							Header: (
+								<StyledTableHeader>
+									{t('futures.market.user.open-orders.table.size')}
+								</StyledTableHeader>
+							),
+							accessor: 'size',
+							Cell: (cellProps: CellProps<any>) => {
+								return <div>{cellProps.row.original.size}</div>;
+							},
+							sortable: true,
+							width: 50,
+						},
+						{
+							Header: (
+								<StyledTableHeader>
+									{t('futures.market.user.open-orders.table.actions')}
+								</StyledTableHeader>
+							),
+							accessor: 'actions',
+							Cell: (cellProps: CellProps<any>) => {
+								return (
+									<div style={{ display: 'flex' }}>
+										<CancelButton
+											onClick={() => {
+												setAction('cancel');
+											}}
+										>
+											{t('futures.market.user.open-orders.actions.cancel')}
+										</CancelButton>
+										{cellProps.row.original.isExecutable && (
+											<EditButton
+												onClick={() => {
+													setAction('execute');
+												}}
+											>
+												{t('futures.market.user.open-orders.actions.execute')}
+											</EditButton>
+										)}
+									</div>
+								);
+							},
+							width: 50,
+						},
+					]}
+				/>
+			</DesktopOnlyView>
+			<MobileOrTabletView>
+				<StyledTable
+					data={data}
+					noResultsMessage={
+						<TableNoResults>{t('futures.market.user.open-orders.table.no-result')}</TableNoResults>
+					}
+					onTableRowClick={(row) => setSelectedOrder(row.original)}
+					columns={[
+						{
+							Header: <StyledTableHeader>Side/Type</StyledTableHeader>,
+							accessor: 'side/type',
+							Cell: (cellProps: CellProps<any>) => (
+								<div>
+									<MobilePositionSide $side={cellProps.row.original.side}>
+										{cellProps.row.original.side}
+									</MobilePositionSide>
+									<div>{cellProps.row.original.orderType}</div>
+								</div>
+							),
+							width: 100,
+						},
+						{
+							Header: <StyledTableHeader>Size</StyledTableHeader>,
+							accessor: 'size',
+							Cell: (cellProps: CellProps<any>) => <div>{cellProps.row.original.size}</div>,
```

Comment:
This size isn't formatted correctly on mobile:
<img width="342" alt="image" src="https://user-images.githubusercontent.com/10401554/182453030-c8e66348-fbbb-4a1a-bbde-f7a0ebbad8e7.png">